packages/react/src/Logo/brand/LogoMuseumWeesp.tsx

```diff
@@ -0,0 +1,45 @@
+      fill="var(--amsterdam-logo-title-color, #EC0000)"
+    />
+    <path
+      className="amsterdam-logo__emblem"
+      d="M0 38.3642L3.6 34.9356L0 31.507L2.4 29.2213L6 32.6499L9.6 29.2213L12 31.507L8.4 34.9356L12 38.3642L9.6 40.6499L6 37.2213L2.4 40.6499L0 38.3642Z"
+      fill="var(--amsterdam-logo-emblem-color, #EC0000)"
```

Comment:
What’s the added value? The CSS class already applies the same custom property.

This also makes the image less portable – it requires the device rendering it to support CSS in the `fill` attribute.

Reply:
If a device support rendering an svg with `fill="#EC0000"` I think it supports CSS. Using variables has the added benefit that you can easily scope those custom properties for example on a dark background with messing with stylesheets. Basically I'm trying to convince you guys you can have a `<style>` tag in your svg so you can support dark mode.

Reply:
This is removed